package.json

```diff
@@ -39,7 +39,7 @@
     "cypress": "^7.6.0",
     "if-env": "^1.0.4",
     "jest": "^24.9.0",
-    "mocha": "^5.2.0",
```

Comment:
I pulled down this branch and ran "yarn install".  It upgraded mocha to 9.1.0 but kept cypress at 7.6.0.  I ran through a basic functionality test of the site and it looked fine. (Also  cypress tests ran fine.) Should cypress remain at 7.6.0 or should we upgrade to 8.3.0 as well?

Reply:
the main goal of this PR was to upgrade `minimist` package that is used by mocha, to do so we upgrade mocha that automatically has upgraded `minimist`. Cypress upgrade was not a part of this task